src/metal.js

```diff
@@ -1,3 +1,21 @@
-// Add Backbone.Metal extensions here
-// 1. Marionette.Class
-// 2. Marionette.Error and Marionette.Deprecate
+// Missing Metal Items
+// 1. Marionette.Class parity
+// 2. Marionette.Deprecate
+
+Marionette.Class = Metal.Class.extend({});
+
+function classify(obj) {
+  return Marionette.Class.extend(
+    _.extend({constructor: obj}, _.omit(obj.prototype, _.keys(Backbone.Events))),
+    _.omit(obj, _.keys(Marionette.Class))
+  );
```

Comment:
It would be good to omit `constructor` and `initialize` as well since those have the potential of getting wrapped with a function for `_super` in browsers with bad `function.toString`